kolibri/plugins/coach/assets/src/vue/coach-exercise-render-page/answer-history/index.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+
+  <div id="answer-history">
+    <h3>{{ $tr('header') }}</h3>
+
+    <ul id="history-list">
+      <template v-for="(attemptLog, index) in reformatedAttemptLogs">
+        <li v-if="index == 0">
+          <p class="item">
+            {{attemptLog.elapse}}
+          </p>
+        </li>
+        <li v-else-if="reformatedAttemptLogs[index -1].elapse != attemptLog.elapse">
+          <p class="item">
+            {{attemptLog.elapse}}
+          </p>
+        </li>
+        <li @click="setSelected(index)" :class="isSeleteced(index)" class="clickable">
+          <div>
+            <mat-svg
+              v-if="attemptLog.hinted"
+              class="item svg-item svg-hint"
+              category="action"
+              name="lightbulb_outline"
+            />
+            <mat-svg
+              v-else-if="attemptLog.correct === 0"
+              class="item svg-item svg-wrong"
+              category="navigation"
+              name="cancel"
+            />
+            <mat-svg
+              v-else
+              class="item svg-item svg-correct"
+              category="action"
+              name="check_circle"
+            />
+            <h3 class="item">
+              {{ $tr('question') }} {{index+1}}
```

Comment:
I don't know the exact syntax offhand, but the point is that values need to interpolated inside the `$tr` function, not using string concatenation